### PR TITLE
Feature/cleanup scripts

### PR DESCRIPTION
**Pull request by AlexAppleManzer**
## Description

Updated dbmanager to include cleanup scripts that are ran every time the container is started up. Also cleaned up johny's migration script and made a readme.

## Checklist

- [ ] Manually tested locally
- [ ] Additional issue created in [docs repo](https://github.com/architus/docs.archit.us/issues) (if necessary)

### CODE REVIEW

File: dbmanager/dbmanager.py [hunk 1]
```diff
@@ -22,20 +23,21 @@ def main():
...
-        run_migrations(to_run)
+    new_migrations = get_new_migrations(current_migration)
+    if new_migrations:
+        print(f"Running migrations: {list(map(os.path.basename, new_migrations))}")
+        run_migration_scripts(new_migrations)
     else:
         print("No migrations to run")
 
+    cleanup_scripts = get_cleanup_scripts()
+    print(f"Running Cleanup Scripts: {list(map(os.path.basename, cleanup_scripts))}")
```
Comment (jazeved0) [inline dbmanager/dbmanager.py]: list comprehension is usually more readable

File: dbmanager/dbmanager.py [hunk 2]
```diff
@@ -45,8 +47,16 @@ def run_migrations(sql_names):
         if exitcode != 0:
             raise Exception(f"oh no, {file_name} failed to execute")
 
+
+def run_migration_scripts(migration_files):
+    """
+    Runs all migration SQL files in the order given
+    """
+    print('ok')
```
Comment (jazeved0) [inline dbmanager/dbmanager.py]: 🤔

File: dbmanager/dbmanager.py [hunk 3]
```diff
@@ -62,23 +72,29 @@ def execute_sql_script(file):
     return process.returncode
 
 
+def get_cleanup_scripts():
+    cleanup_scripts = get_all_files(CLEANUP_DIR, suffix=".sql")
```
Comment (jazeved0) [inline dbmanager/dbmanager.py:81]: Add doc comment >:(

File: dbmanager/dbmanager.py [hunk 4]
```diff
@@ -22,20 +23,21 @@ def main():
...
+    run_scripts(cleanup_scripts)
 
-def run_migrations(sql_names):
-    """
-    Runs all migration SQL files in the order given
-    """
-    for file_name in sql_names:
-        exitcode = execute_sql_script(os.path.join(MIGRATIONS_DIR, file_name))
+
+def run_scripts(file_paths):
```
Comment (jazeved0) [inline dbmanager/dbmanager.py:38]: Add doc comment >:(

File: dbmanager/readme.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+# DB Manager
```
Comment (jazeved0) [inline dbmanager/readme.md]: this file should be all caps `README.md` for consistency

**Review by jazeved0 — CHANGES_REQUESTED** [review]

File: dbmanager/dbmanager.py (hunk 2 above)
Comment (AlexAppleManzer) [reply dbmanager/dbmanager.py]: :)

**Review by jazeved0 — APPROVED** [review]

File: dbmanager/dbmanager.py [hunk 6]
```diff
@@ -45,8 +50,15 @@ def run_migrations(sql_names):
         if exitcode != 0:
             raise Exception(f"oh no, {file_name} failed to execute")
 
+
+def run_migration_scripts(migration_files):
+    """
```
Comment (johnyburd) [inline dbmanager/dbmanager.py:55]: you're allowed to do single lines like `"""comments here"""`

**Review by johnyburd — APPROVED** [review]

**Pull request merged**